src/Core/CMakeLists.txt

```diff
@@ -24,40 +24,33 @@ target_include_directories(ObEngineCore
 )
 
 if(OBE_USE_VCPKG)
+    find_package(fmt CONFIG REQUIRED)
+    find_package(spdlog CONFIG REQUIRED)
+
+    find_package(sol2 CONFIG REQUIRED)
+    find_package(Catch2 CONFIG REQUIRED)
+
+    target_include_directories(ObEngineCore PUBLIC
```

Comment:
Why is this needed ? `target_link_libraries` should add the necessary include paths to the target ?